GameData/RealismOverhaul/RO_RecommendedMods/Procedurals/RO_ProceduralParts.cfg

```diff
@@ -1,7 +1,14 @@
 @PART[proceduralBattery]:FOR[RealismOverhaul]
 {
 	%RSSROConfig = True
-	%title = Battery [Procedural]
+
+	%category = Electrical
+	@title = Battery [Procedural]
```

Comment:
Is there any benefit of using @ over %? % makes it if it doesn't exist, otherwise does an update, correct?  Is % any less efficient?  It would seem like anytime we want to make _sure_ we have something set, we should use %, where @ is "i only want to change this if it has a value"